src/lib.rs

```diff
@@ -100,6 +100,73 @@ pub fn build<P: AsRef<Path>>(path: P) -> PathBuf {
 }
 
 impl Config {
+    /// Infer `CMAKE_BUILD_TYPE` from Rust's compilation profile
+    ///
+    /// * if `opt-level=0` then `CMAKE_BUILD_TYPE=Debug`,
+    /// * if `opt-level={1,2,3}` and:
+    ///   * `debug=false` then `CMAKE_BUILD_TYPE=Release`
+    ///   * otherwise `CMAKE_BUILD_TYPE=RelWithDebInfo`
+    /// * if `opt-level={s,z}` then `CMAKE_BUILD_TYPE=MinSizeRel`
+    pub fn default_build_profile(&self) -> &'static str {
```

Comment:
Could this perhaps be `get_profile` and respect the value in `self.profile` if it's already configured?